docs/data.md

```diff
@@ -24,6 +24,31 @@ the level and type of the log is listed.
 
 ### `merino-web`
 
+- `INFO web.suggest.request` - A suggestion request is being processed. This
+  event will include fields for all relevant details of the request. **Fields:**
+
+  - `query` - If query logging is enabled, the text the user typed. Otherwise
+    and empty string.
+  - `country` - The country the request came from.
+  - `region` - The first country subdivision the request came from.
+  - `city` - The city the request came from.
+  - `dma` - A US-only location description that is larger than city and smaller
+    than states, but does not align to political borders.
+  - `agent` - The original user agent.
+  - `os_family` - Parsed from the user agent. One of "windows", "macos",
+    "linux", "ios", "android", "chrome os", "blackberry", or "other".
+  - `form_factor` - Parsed from the user agent. One of "desktop", "phone",
+    "tablet", or "other"
+  - `browser` - The browser and possibly version detected. Either "Firefox(XX)"
```

Comment:
Where is this UA parsing implemented? Do we have standards for this across Fx services? It would be desirable to have uniform behavior with [the parsing that happens in the telemetry pipeline](https://github.com/mozilla/gcp-ingestion/blob/main/ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUserAgent.java) but is certainly out of scope for this PR. Do we think this is worth writing up an issue for?

Reply:
I added a link to the `DeviceInfo` code inside of `merino-web/src/extrators.rs`. That takes the UA that's parsed by `woothee` and normalizes it down.

Reply:
Getting this aligned may be interesting, but definitely out of scope for this PR. I think the approach we should do is to compare the telemetry that Merino emits and the telemetry pipeline generates, and see how well it matches. If it matches well enough we probably don't need to do anything. If it is too different, we can figure out what to do then.

Also of note: Merino doesn't currently use this information for anything; I've considered disabling it entirely if there are any performance concerns (though #194 makes that less important).